recipes/sos-notebook/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "sos-notebook" %}
+{% set version = "0.19.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9f3ae4371f2a9cf493c77190010e73270ddc658904618937889ce14c0ecebb09
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+    - {{ PYTHON }} -m sos_notebook.install --sys-prefix
+
+requirements:
+  host:
+    - ipython
+    - notebook >=5.0.0
+    - pip
+    - python >=3.6
+  run:
+    - ipykernel
+    - ipython
+    - markdown
+    - matplotlib
+    - nbconvert >=5.1.1
+    - nbformat
+    - notebook >=5.0.0
+    - pandas
+    - python >=3.6
+    - setuptools
+    - sos >=0.19.2
+    - tabulate
+
+test:
+  requires:
+    - nose
+  imports:
+    - sos_notebook
+  commands:
+    - jupyter kernelspec list | grep -w $PREFIX/share/jupyter/kernels/sos  # [unix]
```

Comment:
This does not work under mac here. It seems that `|` split the commands into two.

```
+ jupyter kernelspec list
+ grep -w /Users/bpeng1/anaconda3/conda-bld/sos-notebook_1554269979093/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/share/jupyter/kernels/sos
Tests failed for sos-notebook-0.19.1-py_0.tar.bz2 - moving package to /Users/bpeng1/anaconda3/conda-bld/broken
WARNING:conda_build.build:Tests failed for sos-notebook-0.19.1-py_0.tar.bz2 - moving package to /Users/bpeng1/anaconda3/conda-bld/broken
removing: sos-notebook-0.19.1-py_0.tar.bz2
TESTS FAILED: sos-notebook-0.19.1-py_0.tar.bz2
```

Reply:
@isuruf I would also appreciate your guidance here. Did `|` really separate the command into two? Then the second `grep` command should hang indefinitely.

Reply:
No, it doesn't split the command into two. It's just that when the commands are echoed they are shown like that

Reply:
I see, then I think the 

```
    - {{ PYTHON }} -m sos_notebook.install --sys-prefix
```
command does not belong to `build`, and should be in some section like `install` to be able to run before `test`.

Reply:
Figured out the problem, the build and test steps use different environments with different python.

@jdblischak Please change

```
    - jupyter kernelspec list | grep -w $PREFIX/share/jupyter/kernels/sos  # [unix]
    - jupyter kernelspec list | findstr \share\jupyter\kernels\sos  # [win]
```
to
```
    - jupyter kernelspec list | grep kernels/sos  # [unix]
    - jupyter kernelspec list | findstr kernels\sos  # [win]
```

Reply:
@BoPeng I don't understand your concern. This test passes on macOS, which you can see at the line below:

https://travis-ci.org/conda-forge/staged-recipes/builds/513186637#L1626

Reply:
It failed locally on my mac. I think the problem was that I have locally installed kernels and multiple jupyter commands so `jupyter kernelspec list` returned my local specs. The test machine does not have this problem.

Reply:
Hmmm....that still doesn't seem right. Even though you have local kernels installed, the sos kernel should still be installed in the conda environment, and thus the `grep` command should find it. I'm not sure what is going on. Is the command `python -m sos_notebook.install --sys-prefix` affected by the presence of locally installed kernels? (e.g. maybe it won't install if it already exists locally?)

Reply:
The jupyter command used in the test might be from local environment. The change can be useful because it makes a bit easier to test the recipe locally, even if the wrong jupyter or python was used. Not a big deal though.

Reply:
The jupyter command used in the test might be from local environment. The change can be useful because it makes a bit easier to test the recipe locally, even if the wrong jupyter or python was used. Not a big deal though.

Reply:
> The jupyter command used in the test might be from local environment.

That shouldn't happen. In a conda environment, the first `jupyter` found via `PATH` should always be the one installed as part of the conda environment.

> The change can be useful because it makes a bit easier to test the recipe locally, even if the wrong jupyter or python was used.

If the wrong jupyter or python is being used, it is good that the test fails since this indicates something is wrong with the isolation of the conda environment.